### DISCUSSION TITLE
Foundry- Zksync ( install)

### DISCUSSION THREAD

**Discussion by 0xhung295** [Cyfrin/foundry-full-course-cu#2860] [Q&A] [answered]
Foundry- Zksync ( install)

**Comment by 0xhung295** [comment]
I'm getting a issue when trying install foundry-zksync:
![image](https://github.com/user-attachments/assets/e444aa64-93d9-4a34-941a-252c3fcf0d82)

Anyone have the same problem?Thanks

**Reply by EngrPips** [reply, marked as answer]
Did you follow the installation instructions here https://foundry-book.zksync.io/getting-started/installation ?

**Comment by 5hr1ganesh** [comment]
@0xhung295 you can just clone the repository in your folder, using 
```
git clone https://github.com/matter-labs/foundry-zksync.git
```
 and then run 
```
foundryup-zksync
```

it should get installed

**Reply by EngrPips** [reply]
Thanks for the heads-up

**Reply by 5hr1ganesh** [reply]
@EngrPips what are you ? a bot or something ?

**Reply by EngrPips** [reply]
No, I am not a bot!

**Reply by 0xhung295** [reply]
Thank you @0xhung295 @EngrPips @5hr1ganesh

**Reply by EngrPips** [reply]
Anytime, Friend.

**Comment by codex-scribe** [comment]
@5hr1ganesh 's comment above worked for me.  I was getting the same error. 

The foundry-book website didn't seem to really match the process in the course video, and also installed a ton of packages on my computer, which I don't know what they are


So I recommend _DO NOT_ follow these instructions
![image](https://github.com/user-attachments/assets/99e365ea-8137-46a8-967a-99ab50f9db38)

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out.
